cmd/oras/push.go

```diff
@@ -71,6 +71,9 @@ Example - Push file to the insecure registry:
 
 Example - Push file to the HTTP registry:
   oras push localhost:5000/hello:latest hi.txt --plain-http
+
+Example - Push oras artifact manifest type with no file:
```

Comment:
```suggestion
Example - Push an oras artifact manifest with inlined annotation:
oras push localhost:5000/hello:latest --annotation 
```

If there is no manifest annotation or blob(oras)/file layer(oci) provided , oras should exit with error. 

 @junczhuMSFT is working on the annotation part, please talk with him on this.

Reply:
@junczhuMSFT Let me add a check for valid arguments.